plugins/modules/apt_rpm.py

```diff
@@ -270,7 +270,7 @@ def install_packages(module, pkgspec):
         rc, out, err = module.run_command("%s -y install %s" % (APT_PATH, packages), environ_update={"LANG": "C"})
 
         installed = True
-        for packages in pkgspec:
+        for package in pkgspec:
             if not query_package_provides(module, package):
```

Comment:
It's `package` here, but still `packages` below. I don't know this module and how it works, but is this really correct?

Reply:
`packages` is the concatenated list of elements of `pkgspec`, and below it would be used correctly assuming the variable wouldn't be clobbered here. So from the logic, the one below should stay `packages`, and this one should be `package`.